leptos_dom/src/events.rs

```diff
@@ -70,6 +70,12 @@ pub fn add_event_listener<E>(
 
     let cb = Closure::wrap(cb as Box<dyn FnMut(E)>).into_js_value();
     let key = intern(&key);
+    assert_eq!(
```

Comment:
It think it's worth gating this with `#[cfg(debug_assertions)]` to keep release binaries as small as possible. What do you think?

(Thank you for your work on these PRs!)